src/mongo/MongoScrapbook.ts

```diff
@@ -281,7 +281,9 @@ class FindMongoCommandsVisitor extends MongoVisitor<MongoCommand[]> {
 		} else if (child instanceof ErrorNode) {
 			return {};
 		} else {
-			throw new Error("Unrecognized node type encountered");
+			let err = new Error(`Unrecognized node type encountered. We could not parse ${child.text}`);
+			this.addErrorToCommand(parseError(err), ctx);
```

Comment:
Why are you creating an error then parsing it and only using the message?

Reply:
I didn't want to change a whole lot of code. addErrorToCommand takes in an IParsedError. I'll change the typing for addErrorToCommand to only require the a message field.